Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use vLLM's vllm-openai server image as the base
+FROM vllm/vllm-openai:latest
+
+# Set working directory
+WORKDIR /workspace
```

Comment:
Can we allow user-defined workdir? Maybe we can default to `/workspace` if none is provided.